stubs/flake8-docstrings/flake8_docstrings.pyi

```diff
@@ -0,0 +1,13 @@
+import argparse
+import ast
+from typing import Any, Generator, Type
+
+class pep257Checker:
+    name: str
+    version: str
```

Comment:
These two should be `ClassVar`s. It is also missing instance attributes `tree`, `filename`, `checker` and `source`. You can use `checker: Any`, because we don't have stubs for `pep257`.